### ISSUE TITLE
help in keras ersion

### ISSUE THREAD

**Issue by Mohamedmtc** [AdrianNunez/Fall-Detection-with-CNNs-and-Optical-Flow#3]
please i need your help i cant load the retrained model vgg_16
i have this error at WEIGHT INITIALIZATION

...............................................................................................................................................................................................................
File "<ipython-input-13-341c874f186f>", line 6, in <module>
    layer_dict[layer].W.set_value(w2)
AttributeError: 'Conv2D' object has no attribute 'W'
..............................................................................................................................................................................................................

 when i searched for this error i found this post say

...............................................................................................................................................................................................................
Please read: https://github.com/fchollet/keras/wiki/Keras-2.0-release-notes
Also, code tells you everything. Change to layer.kernel, layer.bias
...............................................................................................................................................................................................................

 and i don't know what i have to do .... please help me with your keras version or editing your code to the last keras

**Comment by heartofdream** [comment]
keras version 2.1.3  ;tensorflow version 1.0.0

**Comment by AdrianNunez** [comment]
Hi,

My code uses Keras 1.2.0 and the Theano backend, switch to this versions to fully replicate my project. If you want to use Keras 2.0, weights are accessed with layer_dict[layer].kernel and biases with layer_dict[layer].biases. However, if you are using Tensorflow backend, to set the weights you must use the set_weights function. Example:

```
layer_dict[layer].set_weights((w2, b2))
```

This line should replace [line 251](https://github.com/AdrianNunez/Fall-Detection-with-CNNs-and-Optical-Flow/blob/master/temporalneturfd.py#L251) and the next one (where you actually set the weights). Take into account too that you need to transpose the order of the dimensions of the weights for the Tensorflow backend. In [line 248](https://github.com/AdrianNunez/Fall-Detection-with-CNNs-and-Optical-Flow/blob/master/temporalneturfd.py#L248) the order is for Theano, Tensorflow however requires:

```
w2 = np.transpose(np.asarray(w2), (2,3,1,0))
```

Also, change [line 249](https://github.com/AdrianNunez/Fall-Detection-with-CNNs-and-Optical-Flow/blob/master/temporalneturfd.py#L249) to this:

```
w2 = w2[::-1, ::-1, :, :]
```

To summarize, the part of the code where the weights are set should look like this:

```
for layer in layerscaffe[:-3]:
        w2, b2 = h5['data'][layer]['0'], h5['data'][layer]['1']
        w2 = np.transpose(np.asarray(w2), (2,3,1,0))
        w2 = w2[::-1, ::-1, :, :]
        b2 = np.asarray(b2)
        layer_dict[layer].set_weights((w2, b2))

layer = layerscaffe[-3]
w2, b2 = h5['data'][layer]['0'], h5['data'][layer]['1']
w2 = np.transpose(np.asarray(w2), (1,0))
b2 = np.asarray(b2)
layer_dict[layer].set_weights((w2, b2))
```

With these changes I could make the temporalneturfd.py start training in Keras 2.0 with Tensorflow backend, although I haven't tested the accuracy obtained with these modifications.

**Comment by heartofdream** [comment]
Thank you for your help.
